### DISCUSSION TITLE
endless "sobridge: port 3" messages

### DISCUSSION THREAD

**Discussion by DJ-3ncoder** [Security-Onion-Solutions/securityonion#11225] [2.4] [answered]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

airgap

### Hardware Specs

Exceeds minimum requirements

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

I set up Security Onion and did a reboot after the "success" message.
Currently no ethernet ports are connected.

On start I get an endless loop of
```
sobridge: port 3(_random numbers and letters_) entered disabled state
sobridge: port 3(_random numbers and letters_) entered blocking state
sobridge: port 3(_random numbers and letters_) entered forwarding state
device _random numbers and letters_ entered promiscuous mode
eth0: renamed from _random numbers and letters_
```

when checking ```sudo so-status```
```System appears to be starting. No highstate has completed since the system was restarted```

I did no additional configuration to the install after the "success" message.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by DJ-3ncoder** [comment]
Even when I connect a device to the management interface "eno3"  it goes on and on and on.

**Comment by TOoSmOotH** [comment, marked as answer]
Those are console messages as you are connected to the console. You can just ssh to the machine or use the command `dmesg -D` to disable them. I personally don't recommend modifying this setting unless you are using a VM for testing.

**Reply by bienio20** [reply]
but this is not issue fixing answer.

**Reply by TOoSmOotH** [reply]
There are no plans to modify the default kernel messages that get pushed to the console. This is not a Security Onion issue. If these messages bother you you can disable them. https://www.google.com/search?q=how+do+I+disable+console+messages+in+linux

**Reply by bienio20** [reply]
This issue also refers to that docker wont work by typing `sudo so-status`
"System appears to be starting. No highstate has completed since the system was restarted"
